src/appleseed/renderer/modeling/aov/aovfactoryregistrar.cpp

```diff
@@ -62,19 +62,23 @@ struct AOVFactoryRegistrar::Impl
     Registrar<IAOVFactory> m_registrar;
 };
 
-AOVFactoryRegistrar::AOVFactoryRegistrar(const SearchPaths& search_paths)
-  : impl(new Impl())
+AOVFactoryRegistrar::AOVFactoryRegistrar()
+  : impl(new Impl()),
+    m_loaded_libraries(boost::make_shared<loaded_libs_container>())
 {
-    reinitialize(search_paths);
+    reinitialize(m_loaded_libraries);
 }
 
 AOVFactoryRegistrar::~AOVFactoryRegistrar()
 {
     delete impl;
 }
 
-void AOVFactoryRegistrar::reinitialize(const SearchPaths& search_paths)
+void AOVFactoryRegistrar::reinitialize(
+    const boost::shared_ptr<loaded_libs_container> loaded_libraries)
 {
+    m_loaded_libraries = loaded_libraries;
```

Comment:
I don't get it: `reinitialize()` is called with `m_loaded_libraries` in argument, and this gets assigned again to `m_loaded_libraries`?